engine/consensus/dkg/processor/engine_test.go

```diff
@@ -0,0 +1,251 @@
+package processor
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"gotest.tools/assert"
+
+	"github.com/onflow/flow-go/crypto"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module/dkg"
+	module "github.com/onflow/flow-go/module/mock"
+	"github.com/onflow/flow-go/network/mocknetwork"
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+// TestImplementsDKGProcessor ensures that Engine implements the DKGProcessor
+// interface of the crypto package.
+func TestImplementsDKGProcessor(t *testing.T) {
+	var _ crypto.DKGProcessor = (*Engine)(nil)
+}
+
+// TestPrivateSend_Valid checks that the processor correctly converts the
+// message destination parameter (index in committee list) to the corresponding
+// public Identifier, and successfully sends a DKG message to the intended
+// recipient through the network conduit.
+func TestPrivateSend_Valid(t *testing.T) {
+
+	// initialize the list of dkg participants
+	committee := unittest.IdentifierListFixture(2)
+
+	// define epoch ID
+	epochID := rand.Uint64()
```

Comment:
`math/rand` is always initialized by the same seed, so it's better to call `rand.Seed(time.Now().UnixNano())` once for each test to randomize the test.

Reply:
Well it's not very important here as only the epoch ID is random which doesn't really impact the tests.